hack/lib/build/binaries.sh

```diff
@@ -219,6 +219,17 @@ os::build::internal::build_binaries() {
         local_ldflags+=" -s"
       fi
 
+      #Add Windows File Properties/Version Info and Icon Resource for oc.exe
+      if [[ $platform == "windows/amd64" ]]; then
+        os::build::version::get_vars
```

Comment:
I follow the code [here](https://github.com/openshift/origin/blob/master/hack/lib/build/binaries.sh#L440), and when I remove this line, it throws `src/github.com/openshift/origin/hack/lib/build/binaries.sh: line 225: OS_GIT_MAJOR: unbound variable`